source/guide_statping.rst

```diff
@@ -0,0 +1,154 @@
+.. author:: Moanos <https://hyteck.de/about/>
+
+.. tag:: monitoring
+.. tag:: web
+.. tag:: self-hosting
+.. tag:: prometheus
+.. tag:: go
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/statping.png
+      :align: center
+
+########
+Statping
+########
+
+.. tag_list::
+
+
+`Statping`_ is a lightweight and easy to install monitoring tool written in Go.
+It is published under the GNU General Public License v3.0.
+
+
+----
+
+.. note:: For this guide you need some tools:
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+Prerequisites
+=============
+
+Set up the backends:
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 8080
+  Set backend for / to port 8080; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+Create a directory for the configuration.
+
+::
+
+  [isabell@stardust ~]$ mkdir ~/statping
+
+
+Create a database
+
+::
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE eliza_statping"
+
+
+Installation
+============
+
+Now let's get started with installing Statping .
+
+Start with cloning the Statping binaries from Github_ and be sure to replace the version ``v0.80.70`` with the current release number!:
+
+::
+
+ [isabell@stardust ~]$ cd statping
+ [isabell@stardust statping]$ curl -sL https://github.com/statping/statping/releases/download/v0.80.70/statping-linux-x64.tar.gz | tar xz
+ [isabell@stardust statping]$ ls
+ statping  statping.asc
+
+You can now verify the GPG signature in statping.asc
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/statping.ini`` with the following content:
+
+.. code-block:: ini
+
+   [program:statping]
+   directory=%(ENV_HOME)s/statping/
+   command=%(ENV_HOME)s/statping/statping
+   autostart=yes
+   autorestart=yes
+
+Now let's start the service:
+
+.. include:: includes/supervisord.rst
+
+
+Configuration
+=============
+
+You can now access Statping via your browser by visiting <username>.uber.space or any domain
+you set up.
+
+.. _fig_statping_config:
+
+.. figure:: ./_static/images/statping_config.png
+   :align: center
+
+   Configuration site of statping.
+
+Select the Database Connection :manual:`MYSQL <database-mysql>`  and type in your your username and password.
+The database runs on localhost, the port is 3306.
+
+If you don't remember your password use
+
+.. code-block:: console
+
+   [eliza@dolittle ~]$ my_print_defaults client
```

Comment:
There is a snippet for mysql, which is usually included [near the top](https://lab.uberspace.de/guide_etherpad.html#prerequisites). Do you mind using that one?

```
.. include:: includes/my-print-defaults.rst
```